package.json

```diff
@@ -46,6 +46,7 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^14.2.0",
+    "enzyme": "^3.11.0",
```

Comment:
Why did you install `enzyme`? 🤔

Reply:
try to test the onChange function, but it did not working.

Reply:
You won't need `enzyme` for that :) enzyme is another test runner like jest.

Reply:
I try to test the onChange function. One way I found, needs enzyme, but it did not work for me.